src/en/charms-upgrading.md

```diff
@@ -56,6 +56,15 @@ so long as the following conditions are met:
 
 The new charm may add new relations and configuration settings.
 
+## Forced upgrades
+
+A charm upgrade may require the use of the `--force-series` option.
+
+Consider the case where an application is initially deployed using a charm that
+supports Trusty and Precise. If a new version of the charm is released that
+only supports Trusty and Xenial then applications deployed on Precise will only
```

Comment:
Several things trouble me about this paragraph:
 Surely, "Precise and Trusty". Why wouldn't you put them in chronological order?
i don't understand 'applications on precise will only be allowed by...'
What series, in this scenario, was the application deployed on?
What series are you trying to move it to? 
It isn't very clear or easy to follow

Reply:
Yes, I changed the order of Trusty and Precise due to a brain fart.
Re your other comments, I neglected a couple of words. It should read better now.